commands/build/lib/build.js

```diff
@@ -86,7 +86,7 @@ const config = ({
   let pkg = require(path.resolve(CWD, 'package.json')); // eslint-disable-line
   const corePkg = core ? require(path.resolve(core, 'package.json')) : null; // eslint-disable-line
   pkg = reactNative ? require(path.resolve(reactNativePath, 'package.json')) : pkg; // eslint-disable-line
-  const { sourcemap, replacementStrings = {}, typescript } = argv;
+  const { sourcemap, replacementStrings = {}, typescript, disableSourceMaps, preferBuiltins, browser } = argv;
```

Comment:
Can you add these options and descriptions/explanations in the README.md file as well?
And in `commands/build/lib/init-config.js` as well.

Reply:
Done